cluster-build-gsslab.sh

```diff
@@ -1,33 +1,79 @@
 #!/bin/bash
 
 DEFAULT_OCPVERSION=4.3.8
+CLUSTER_SIZE=small
+WORKER_MEMORY=8192
+WORKER_CPU=2
+DISCONNECTED=false
+OCS_SETTING=false
+BUILD_LAB=gsslab
```

Comment:
We should have one build script then and not `cluster-build-gsslab` etc.

Reply:
I agree, that's the next task